LifeCycleAssessment_Engine/Query/ProjectLifeCycleAssessmentResults.cs

```diff
@@ -0,0 +1,166 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.LifeCycleAssessment.Results;
+using BH.oM.Reflection.Attributes;
+using System;
+using System.ComponentModel;
+using System.Linq;
+
+namespace BH.Engine.LifeCycleAssessment
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****   Public Methods                          ****/
+        /***************************************************/
+
+        [Description("Query the standard project results from a complete ProjectLifeCycleAssessment evaluation for pushing to datasets.")]
+        [Input("lcaResult", "Supply a complete LifeCycleAssessmentResult to query.")]
+        [Output("projectResults", "The combined amount of kgCO2e of the objects provided.")]
+        public static ProjectLifeCycleAssessmentResult ProjectLifeCycleAssessmentResult(this LifeCycleAssessmentResult lcaResult)
+        {
+            double buildingLifespan = lcaResult.LifeCycleAssessmentScope.BuildingLifespan;
+            if (double.IsNaN(buildingLifespan))
+            {
+                BH.Engine.Reflection.Compute.RecordNote("No building lifespan has been provided for this entry. Returning zero.");
+                buildingLifespan = 0;
```

Comment:
You say returning 0, but then allow the calculation to continue? Is this intended?

Reply:
Yes this is intended and preferred IMO to setting a default value that may not be true. At this point in time there are no calculations depending on this property and it is used exclusively for reporting purposes.

Reply:
I think the issue here was more on the note that may not return 0 from the method. I wonder if it would be better for the note to say `Using zero as building lifespan` to be more accurate of what the method is doing?